SU-HSDO/suhumsci
# shs-94: fix accordion float bug

## Summary
This fixes a bug in the accordion when an image is floated in its container.

## Steps to Test
1. Go to the Colorful demo site. Visit the accordion component page: `/components/accordions`. 
2. Click on the one near the top with 'Accordion with Image and Text'. When open, the image should be within the container (like the screenshot below).
3. Do the same for the Traditional site and make sure the other accordions behave as normal

## Screenshots
![Screen Shot 2023-01-04 at 9 31 54 AM](https://user-images.githubusercontent.com/8405274/210578029-50f08c6b-2137-47fb-8553-a427019c93bd.png)

## Notes
The accordion is actually controlled by the `hs_layouts` module, so I created an override stylesheet.
